remote-helloworld-mdb/pom.xml

```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    JBoss, Home of Professional Open Source
+    Copyright 2015, Red Hat, Inc. and/or its affiliates, and individual
+    contributors by the @authors tag. See the copyright.txt in the
+    distribution for a full listing of individual contributors.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.wildfly.quickstarts</groupId>
+        <artifactId>wildfly-quickstart-parent</artifactId>
+        <!--
+        Maintain separation between the artifact id and the version to help prevent
+        merge conflicts between commits changing the GA and those changing the V.
+        -->
+        <version>2</version>
+        <relativePath/>
+    </parent>
+
+    <artifactId>remote-helloworld-mdb</artifactId>
+    <version>28.0.0.Beta1-SNAPSHOT</version>
+    <packaging>war</packaging>
+    <name>Quickstart: remote-helloworld-mdb</name>
+    <description>This project demonstrates a hello world Message-Driven Bean with Servlet 6.0 as client to a remote Artemis instance</description>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.html</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <properties>
+        <!-- The versions for BOMs, Dependencies and Plugins -->
+        <version.server.bom>27.0.0.Final</version.server.bom>
+        <version.server>27.0.0.Final</version.server>
+        <version.wildfly.maven.plugin>4.0.0.Final</version.wildfly.maven.plugin>
+        <version.cloud.fp>2.0.1.Final</version.cloud.fp>

```

Comment:
All other migrated quickstarts have 2.0.0.Final https://github.com/wildfly/quickstart/search?q=%3Cversion.cloud.fp%3E

Would it work with the 2.0.0.Final version so that it is synced with the rest of the repo (and thus avoiding potential missing during mass update)?